incubator/kafka/templates/statefulset.yaml

```diff
@@ -25,7 +25,7 @@ spec:
             command:
               - bin/kafka-topics.sh
               - --zookeeper
-              - "{{ template "name" . }}-zookeeper:2181"
+              - "{{ .Release.Name }}-zookeeper:2181"
```

Comment:
Just curious Have you tried `{{ template "fullname" }}` instead? does this still work if you install this release without explicitly giving it a name using `--name` argument?